android/app/src/main/java/io/parity/signer/components/HistoryCardExtended.kt

```diff
@@ -37,114 +40,120 @@ fun HistoryCardExtended(card: JSONObject) {
 				danger = true
 			)
 		}
-		"general_verifier_added" -> {
+		is Event.GeneralVerifierSet -> {
+			val hex = eventVal.verifier.v.let {
+				when (it) {
+					is VerifierValue.Standard -> {
```

Comment:
This is not right. General verifier could only be *set* to `Standard`, this should not be allowed to be optional, even on rust side.

Reply:
My wrong, same event sets it to none on wipe-general-verifier reset